src/BenchmarksDriver/Program.cs

```diff
@@ -444,6 +454,7 @@ public static int Main(string[] args)
                         [ConnectionFilter] [nvarchar](max) NULL,
                         [WebHost] [nvarchar](max) NOT NULL,
                         [KestrelThreadCount] [int] NULL,
+                        [KestrelThreadPoolDispatching] [bit] NULL,
```

Comment:
`null` and `true` have exactly the same meaning once Kestrel starts up. But :ok: